File: Java/3311.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        String[][] namesList = new String[26][n];
        int[] namesIndex = new int[26];
        for (int i = 0; i < n; i++) {
            String name = scanner.next();
            int index = Character.toLowerCase(name.charAt(0)) - 'a';
            namesList[index][namesIndex[index]++] = name;
        }
        for (String[] names : namesList) {
            if (names != null) {
                for (String name : names) {
                    if (name != null) {
                        System.out.println(name);
                    }
                }
            }
        }
    }
}

File: Python/3311.py
n = int(input())
names_list = [[] for _ in range(26)]

for _ in range(n):
    name = input()
    index = ord(name[0].lower()) - ord('a')
    names_list[index].append(name)

for names in names_list:
    for name in names:
        if name:
            print(name)